server/webhook.go

```diff
@@ -72,164 +72,156 @@ func (s *Server) registerWebhookRoutes(g *echo.Group) {
 			zap.String("project", repo.Project.Name),
 		)
 
+		distinctFileList := dedupMigrationFilesFromCommitList(pushEvent.CommitList)
 		createdMessageList := []string{}
-		for _, commit := range pushEvent.CommitList {
-			log.Debug("Processing commit...",
-				zap.String("id", common.EscapeForLogging(commit.ID)),
-				zap.String("title", common.EscapeForLogging(commit.Title)),
+		for _, item := range distinctFileList {
+			commit := item.commit
+			added := item.fileName
+			addedEscaped := common.EscapeForLogging(added)
+			log.Debug("Processing added file...",
+				zap.String("file", addedEscaped),
+				zap.String("commit", commit.ID),
```

Comment:
Consider using `common.EscapeForLogging(commit.ID)` to prevent logging injection and make the CodeQL scanner happy.

Reply:
Done